packages/redux/src/recommendations/__tests__/reducer.test.ts

```diff
@@ -1,11 +1,13 @@
-import { mockRecommendationsStrategy } from '../../../tests/__fixtures__/initialReduxState';
+import { INITIAL_STATE } from '../reducer';
+import { mockRecommendationsStrategy } from 'tests/__fixtures__/recommendations';
 import reducer, { actionTypes } from '..';
+import type { State } from '../types';
 
 let initialState;
-
+const mockAction = { type: 'this_is_a_mock_action' };
 describe('recommendations reducer', () => {
   beforeEach(() => {
-    initialState = reducer();
+    initialState = reducer(INITIAL_STATE, mockAction);
```

Comment:
INITIAL_STATE should be undefined to make sure the reducer is initialised with the INITIAL_STATE.

Reply:
Source updated, can you check again.